.gitignore

```diff
@@ -41,3 +41,7 @@ dist/
 
 .DS_Store
 *.work
+
+# dotnet
+*.sln
```

Comment:
Is there any particular reason to ignore sln files?

Reply:
But I think a `solution` is an artifact that groups multiple projects together. In this case we have multiple projects the actual sdk and the other examples. 
A solution file might be important to add because we could build and execute examples at `sdk-dotnet` root path.

Reply:
but we do not have a solution with different projects, we have a folder with different dotnet projects, until now I have not seen the usage of the sln file, vscode adds it in each proj but not need in command line

Reply:
What I'm saying is that you can do something like this:

```bash
dotnet new sln --name Littlehorse 
dotnet sln add LittleHorse.Sdk
dotnet sln add Examples/BasicExample
dotnet build
```

this is a way to build the binaries for the sdk and examples.

Reply:
we do not need because you can have examples where you want event client wont be part of the solution, we just have a SDK project published to public

Reply:
I think we will need this at some point, but it is ok. We can add it in the future if need it.